data/mllib/sample_isotonic_regression_data.csv

```diff
@@ -0,0 +1,101 @@
+4710.28,500.00,1.00
```

Comment:
Btw, we can use `.txt` instead of `.csv` to be consistent with other files in the same folder.